core/sessions/user.go

```diff
@@ -43,8 +43,12 @@ func NewUser(email, plainPwd string) (User, error) {
 		return User{}, errors.New("Invalid email format")
 	}
 
-	if len(plainPwd) < 8 || len(plainPwd) > MaxBcryptPasswordLength {
-		return User{}, fmt.Errorf("must enter a password with 8 - %v characters", MaxBcryptPasswordLength)
+	if len(plainPwd) > MaxBcryptPasswordLength {
+		return User{}, fmt.Errorf("max length is %v characters", MaxBcryptPasswordLength)
```

Comment:
Would be great to reflect the max length in the doc along with the new requirements.